common/src/main/aidl/com/microsoft/identity/common/internal/broker/IBrokerAccountService.aidl

```diff
@@ -0,0 +1,18 @@
+package com.microsoft.identity.common.internal.broker;
+
+ /**
+  * Broker Account service APIs provided by the broker app. Those APIs will be responsible for interacting with the
+  * account manager API. Calling app does not need to request for contacts permission if the broker installed on the
+  * device has the support for the bound service.
+  */
+interface IBrokerAccountService {
```

Comment:
Can we call this something else, so that we are not confused when talking about bug fixes and looking at logs. Lets take Daniels input on what to call this.